Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,14 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/tag-4.9.25.tar.gz"
```

Comment:
Use https://github.com/htacg/tidy-html5/archive/4.9.26.tar.gz which should fix the manpage issue.